extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -106,8 +109,10 @@ define('ext.wikia.adEngine.video.googleIma', [
 				this.events[eventName].push(callback);
 			},
 			playVideo: function (width, height) {
+				console.log('***** IMA PLAY VIDEO');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)